### PR TITLE
Add json log formatter

### PR DESCRIPTION
**Pull request by ivoreis**
This PR enables json logger format:
```
{"level":"debug","msg":"Engine terminated cleanly","time":"2017-12-20T12:30:35Z"}
```
instead of:
```
DEBU[0010] Engine terminated cleanly
```
Usage cmd:
```
k6 run samples/http_get.js --logformat json
```

This can be extended to support other different loggers:
```
k6 run samples/http_get.js --logformat xml
...
```

### CODE REVIEW

**Comment by liclac** [comment]
This honestly should be done in `PersistentPreRun` in `root.go`, along with the verbosity. And let's not make a whole package for that; just instantiate a JSONFormatter right then and there.

**Comment by ivoreis** [comment]
Makes sense. I've pushed the changes.

File: cmd/root.go [hunk 1]
```diff
@@ -95,7 +91,25 @@ func init() {
 	RootCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false, "enable debug logging")
 	RootCmd.PersistentFlags().BoolVarP(&quiet, "quiet", "q", false, "disable progress updates")
 	RootCmd.PersistentFlags().BoolVar(&noColor, "no-color", false, "disable colored output")
+	RootCmd.PersistentFlags().StringVarP(&logFmt, "logformat", "f", "", "log output format")
```
Comment (robingustafsson) [inline cmd/root.go]: Could we make do without the shorthand `-f` here for now and start with only `--logformat`?

Comment (ivoreis) [reply cmd/root.go]: Just pushed the change

**Comment by robingustafsson** [comment]
@ivoreis Great, thanks! Merging.

**Pull request merged**